index.js

```diff
@@ -11,28 +11,29 @@ module.exports = {
     sourceType: 'module',
   },
   rules: {
-    'prettier/prettier': [
-      'error',
-      {
-        singleQuote: true,
-        bracketSpacing: false,
-        trailingComma: 'es5',
-        arrowParens: 'avoid',
-      },
-    ],
     'array-callback-return': 'error',
     'block-scoped-var': 'error',
     curly: 'error',
     'default-case': 'error',
     'dot-notation': ['error', {allowPattern: '^[a-z]+(_[a-z]+)+$'}],
     eqeqeq: 'error',
-    'import/no-unresolved': ['error', {commonjs: true}],
-    'import/named': 'error',
     'import/default': 'error',
+    'import/named': 'error',
+    'import/no-unresolved': ['error', {commonjs: true}],
+    'import/order': [
+      'error',
+      {
+        groups: [
+          ['builtin', 'external'],
+          ['internal', 'parent', 'sibling', 'index'],
+        ],
```

Comment:
There would be far less churn in all projects that use this config if we didn't enforce this grouping.

Reply:
Shall I move this out to a distinct PR so that we can look at it together while the rest can go in already? I really like this rule as it makes the imports very structured - but happy to discuss this in a larger group while the rest can already move in paralell

Reply:
Yes, it would be nice to move the rule without introducing complex new grouping.  We get the benefits of stable sorting whether we have a simple rule (follow the alphabet) or a complex rule (not sure how to succinctly describe the proposed grouping).

Reply:
https://github.com/planetlabs/eslint-config-planet/pull/38